client.go

```diff
@@ -45,7 +45,7 @@ type Client interface {
 
 	// Stop a running command. ErrNotFound is returne if Wait or Stop has already
 	// been called.
-	Stop(id string) (*pb.Status, error)
+	Stop(id string) (*pb.Empty, error)
```

Comment:
Do we just need to remove the 1st return value, i.e. return only `error`?

Reply:
No, protobufs don't support the idea of having "nothing to return." In order to return a NULL response, the Empty{} struct is used. https://godoc.org/github.com/golang/protobuf/ptypes/empty

Reply:
Ah I see what you mean now. Yeah this can be removed, the user doesn't need this.